tests/test_tchannel.py

```diff
@@ -296,3 +296,25 @@ def test_listen_duplicate_ports():
     port = int(server.hostport.rsplit(":")[1])
     server.listen(port)
     server.listen()
+
+
+@pytest.mark.gen_test
+def test_tll():
+
+    server = TChannel(name='server')
+    server.listen()
+
+    tchannel = TChannel(name='client')
+
+    with mock.patch(
+        "tchannel.tornado.dispatch.RequestDispatcher.handle", autospec=True
+    ) as mock_handle:
+        with pytest.raises(TimeoutError):
+            yield tchannel.call(
+                scheme=schemes.RAW,
+                service='server',
+                arg1='endpoint',
+                hostport=server.hostport,
+                timeout=9,
+            )
+        assert mock_handle.call_args[0][1].ttl == 9
```

Comment:
No, you need to check the `ttl` set on the `CallRequestMessage`. That needs to be in milliseconds. Checking on the `Request` object isn't good enough because we could be consistent there while still leaving the `CallRequestMessage` one incorrect -- and that's the one Hyperbahn cares about.